src/Geta.404Handler/Core/CustomRedirects/CustomRedirectCollection.cs

```diff
@@ -144,7 +144,7 @@ string RemoveSlash(string s)
             }
 
             var redirCopy = new CustomRedirect(cr);
-            var newUrl = AppendSlash(redirCopy.NewUrl);
+            var newUrl = (url.IndexOf("?", StringComparison.Ordinal) > 0) ? redirCopy.NewUrl : AppendSlash(redirCopy.NewUrl);
```

Comment:
Please remove unnecessary brackets.
```suggestion
            var newUrl = url.IndexOf("?", StringComparison.Ordinal) > 0 ? redirCopy.NewUrl : AppendSlash(redirCopy.NewUrl);
```